zetaclient/evm/evm_client.go

```diff
@@ -371,6 +372,10 @@ func (ob *ChainClient) WatchOutTx() {
 					ob.SetTxNReceipt(nonceInt, outtxReceipt, outtx)
 				} else if txCount > 1 { // should not happen. We can't tell which txHash is true. It might happen (e.g. glitchy/hacked endpoint)
 					ob.logger.OutTx.Error().Msgf("WatchOutTx: confirmed multiple (%d) outTx for chain %d nonce %d", txCount, ob.chain.ChainId, nonceInt)
+				} else {
+					if len(tracker.HashList) == crosschainkeeper.MaxOutTxTrackerHashes {
+						ob.logger.OutTx.Error().Msgf("WatchOutTx: outbound tracker is full of invalid hashes for chain %d nonce %d", ob.chain.ChainId, nonceInt)
```

Comment:
we are not sure of the "invalid" part right?

Reply:
Yeah, we're not 100% sure, potentially.

Reply:
To make this easier to parse the chainId and nonce should be attributes of the log message instead of embedded in the log message. These can be resued by other logs to so if we want to find all errors associated with a specific nonce it is very easy.